Plugin/UndyingRagdoll.cs

```diff
@@ -288,7 +288,7 @@ private void Update()
                 {
                     if (node.sliced_off)
                     {
-                        rp.rb.isKinematic = false;
+                        rp.GetComponent<Rigidbody>().isKinematic = false;

```

Comment:
Just updated. Thanks for the in-depth explanation! It didn't seem to be an issue but like you said, best to avoid potential bugs and unexpected behavior.